src/main/java/project/common/PrintText.java

```diff
@@ -0,0 +1,90 @@
+package project.common;
+
+/**
+ * All the {@code String} constants commonly used in the application.
+ */
+public enum PrintText {
```

Comment:
For enum name PrintText, perhaps you could change it to something like ‘PrintedText’ to make it sound more like a noun?

Reply:
Right I hadn't noticed that, thanks!